components/ExploreTabs.tsx

```diff
@@ -8,13 +8,15 @@ import BiospecimenTable from './BiospecimenTable';
 import CaseTable from './CaseTable';
 import FileTable from './FileTable';
 import WPAtlasTable from './WPAtlasTable';
+import { DataSchemaData } from '../lib/dataSchemaHelpers';
 
 interface IExploreTabsProps {
     router: NextRouter;
     filteredFiles: Entity[];
     samples: Entity[];
     cases: Entity[];
     wpData: WPAtlas[];
+    schemaDataMap?: { [id: string]: DataSchemaData };
```

Comment:
can the name be made any better? `schemaDataMap` is kind of confusing. for that matter, maybe it would be good to rename `DataSchemaData`, unless other people don't find that confusing? what is schema data?

Reply:
Data model and most of the utility functions were copied from the `Synapse-React-Client` repo. 

https://github.com/Sage-Bionetworks/Synapse-React-Client/blob/01fb7e594517f9f4b4e9950fd0bd21e45e17eeb9/src/lib/containers/data_schema/types/IDataSchemaTypes.ts#L34

We should probably keep `DataSchemaData` for now however it sounds confusing. We can rename `schemaDataMap` to something better though. Maybe something like `schemaDataBySchemaId`?